personal-practice/car.java

```diff
@@ -0,0 +1,37 @@
+import java.awt.*;
+import java.awt.Color;
+
+public class car {
+
+    // Data Types: 
+    // int
+    // double (float)
+    // string
+    // Color (from awt library)
+    // boolean
```

Comment:
This comment block does not look useful. It looks like commented out code which is discouraged.